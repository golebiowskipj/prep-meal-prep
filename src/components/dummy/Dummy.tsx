import React from "react";
import { DummyStyled } from "./styled";

export const Dummy: React.FC = () => (
    <DummyStyled>I am Dummy component</DummyStyled>
);
