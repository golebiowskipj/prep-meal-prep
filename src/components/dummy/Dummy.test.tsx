import React from "react";
import { shallow } from "enzyme";
import { Dummy } from "./Dummy";

describe("<Dummy/>", () => {
    it("matches snapshot", () => {
        expect(shallow(<Dummy />)).toMatchSnapshot();
    });
});
