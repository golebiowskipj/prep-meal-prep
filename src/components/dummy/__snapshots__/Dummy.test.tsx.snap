// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Dummy/> matches snapshot 1`] = `
<styled.h1>
  I am Dummy component
</styled.h1>
`;
